app/src/pages/instance/allUsersPage/allUsersListTable/allUsersListTable.jsx

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import classNames from 'classnames/bind';
+import { AbsRelTime } from 'components/main/absRelTime';
+import { MeatballMenuIcon, Popover, Tooltip } from '@reportportal/ui-kit';
+import { InstanceUserAvatar } from 'pages/instance/common/instanceUserAvatar';
+import { userInfoSelector } from 'controllers/user';
+import { getRoleBadgesData } from 'common/utils/permissions/getRoleTitle';
+import { ADMIN_TYPE } from 'common/utils/permissions/constants';
+import {
+  DEFAULT_PAGE_SIZE,
+  DEFAULT_PAGINATION,
+  PAGE_KEY,
+  withPagination,
+} from 'controllers/pagination';
+import { SORTING_ASC, withSortingURL } from 'controllers/sorting';
+import {
+  DEFAULT_SORT_COLUMN,
+  allUsersPaginationSelector,
+  fetchAllUsersAction,
+} from 'controllers/instance/allUsers';
+import { MembersListTable } from 'pages/organization/common/membersPage/membersListTable';
+import { messages } from 'pages/organization/common/membersPage/messages';
+import styles from './allUsersListTable.scss';
+
+const cx = classNames.bind(styles);
+const NAMESPACE = 'allUsers';
```

Comment:
Pls use `NAMESPACE` from selectors instead.

Reply:
Replaced the local const NAMESPACE with the imported NAMESPACE from constants.js instead of defining it locally